x-pack/plugins/ml/server/lib/alerts/register_anomaly_detection_alert_type.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  ML_ALERT_TYPES,
+  ML_ALERT_TYPES_CONFIG,
+  AnomalyScoreMatchGroupId,
+} from '../../../common/constants/alerts';
+import { PLUGIN_ID } from '../../../common/constants/app';
+import { MINIMUM_FULL_LICENSE } from '../../../common/license';
+import {
+  MlAnomalyDetectionAlertParams,
+  mlAnomalyDetectionAlertParams,
+} from '../../routes/schemas/alerting_schema';
+import { RegisterAlertParams } from './register_ml_alerts';
+import { InfluencerAnomalyAlertDoc, RecordAnomalyAlertDoc } from '../../../common/types/alerts';
+import {
+  AlertInstanceContext,
+  AlertInstanceState,
+  AlertTypeState,
+} from '../../../../alerts/common';
+
+const alertTypeConfig = ML_ALERT_TYPES_CONFIG[ML_ALERT_TYPES.ANOMALY_DETECTION];
+
+export type AnomalyDetectionAlertContext = {
+  name: string;
+  jobIds: string[];
+  timestampIso8601: string;
+  timestamp: number;
+  score: number;
+  isInterim: boolean;
+  topRecords: RecordAnomalyAlertDoc[];
+  topInfluencers?: InfluencerAnomalyAlertDoc[];
+  anomalyExplorerUrl: string;
+} & AlertInstanceContext;
+
+export function registerAnomalyDetectionAlertType({
+  alerts,
+  mlSharedServices,
+}: RegisterAlertParams) {
+  alerts.registerType<
+    MlAnomalyDetectionAlertParams,
+    AlertTypeState,
+    AlertInstanceState,
+    AnomalyDetectionAlertContext,
+    AnomalyScoreMatchGroupId
+  >({
+    id: ML_ALERT_TYPES.ANOMALY_DETECTION,
+    name: alertTypeConfig.name,
+    actionGroups: alertTypeConfig.actionGroups,
+    defaultActionGroupId: alertTypeConfig.defaultActionGroupId,
+    validate: {
+      params: mlAnomalyDetectionAlertParams,
+    },
+    actionVariables: {
+      context: [
+        {
+          name: 'timestamp',
+          description: i18n.translate('xpack.ml.alertContext.timestampDescription', {
+            defaultMessage: 'Timestamp of the anomaly',
+          }),
+        },
+        {
+          name: 'timestampIso8601',
+          description: i18n.translate('xpack.ml.alertContext.timestampIso8601Description', {
+            defaultMessage: 'Time in ISO8601 format',
+          }),
+        },
+        {
+          name: 'jobIds',
+          description: i18n.translate('xpack.ml.alertContext.jobIdsDescription', {
+            defaultMessage: 'List of job ids triggered the alert instance',
```

Comment:
```suggestion
            defaultMessage: 'List of job IDs triggered the alert instance',
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/89286/commits/4acf5401e6441e83ceb4cc237dda2a1b46fe91b2